src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1526,6 +1526,12 @@ public override BoundNode VisitDeconstructionAssignmentOperator(BoundDeconstruct
             return null;
         }
 
+        public override BoundNode VisitDeconstructionVariables(BoundDeconstructionVariables node)
+        {
+            Debug.Assert(false, "BoundDeconstructionVariables should not make it past binding stage.");
+            return null;
```

Comment:
And seal the method.

---
In reply to: [65786461](https://github.com/dotnet/roslyn/pull/11715#discussion_r65786461) [](ancestors = 65786461)